normandy/control/static/control/js/components/RecipeList.jsx

```diff
@@ -1,41 +1,41 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { push } from 'react-router-redux'
+import { Table, Thead, Th, Tr, Td, applyFilter } from 'reactable'
+import classNames from 'classnames'
 import moment from 'moment'
 import ControlActions from '../actions/ControlActions.js'
 
-class BooleanIcon extends React.Component {
-  render() {
-    switch(this.props.value) {
-      case true: return <i className="fa fa-lg fa-check green">&nbsp;</i>;
-      case false: return <i className="fa fa-lg fa-times red">&nbsp;</i>;
-    }
+const BooleanIcon = (props) => {
+  switch(props.value) {
+    case true: return <i className="fa fa-lg fa-check green">&nbsp;</i>;
+    case false: return <i className="fa fa-lg fa-times red">&nbsp;</i>;
   }
 }
 
-class RecipeDataRow extends React.Component {
-  render() {
-    const { recipe, dispatch } = this.props;
-
-    return (
-      <tr key={recipe.id} onClick={(e) => {
-        dispatch(ControlActions.setSelectedRecipe(recipe.id));
-        dispatch(push(`/control/recipe/${recipe.id}/`))
-      }}>
-        <td>{recipe.name}</td>
-        <td>{recipe.action}</td>
-        <td><BooleanIcon value={recipe.enabled} /></td>
-        <td><BooleanIcon value={recipe.is_approved} /></td>
-        <td>{ moment(recipe.last_updated).fromNow() }</td>
-      </tr>
-    )
-  }
+const SwitchFilter = (props) => {
+  const { options, selectedFilter, updateFilter } = props;
```

Comment:
You can put this destructuring right in the function arguments instead of `props`.